ietf-microwave-radio-link.yang

```diff
@@ -142,16 +140,24 @@ module ietf-microwave-radio-link {
       description
         "True when the profile can be used in single
          coding-modulation-mode.";
+    }
+	leaf support-as reference-modulation {
+      type boolean;
+      description
+        "True when the profile can be used as reference-modulation in adaptive
+         coding-modulation-mode.
+		 Reference Modulation cannot be configured but can be set by the system 
+		 when support-as-reference-modulation is false.";
```

Comment:
Reformatted the indentation: please check use of TAB characters instead of spaces in the editor

```suggestion
}
  leaf support-as reference-modulation {
    type boolean;
    description
      "True when the profile can be used as reference-modulation in
       adaptive coding-modulation-mode.

       Reference Modulation cannot be configured but can be set by
       the system when support-as-reference-modulation is false.";
```